app/src/main/java/ca/marshallasch/veil/MemoryStore.java

```diff
@@ -0,0 +1,521 @@
+package ca.marshallasch.veil;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Pair;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import ca.marshallasch.veil.exceptions.TooManyResultsException;
+import ca.marshallasch.veil.proto.DhtProto;
+import ca.marshallasch.veil.utilities.Util;
+
+import static ca.marshallasch.veil.proto.DhtProto.KeywordType.*;
+
+/**
```

Comment:
Add in overview for what the class is for.

Reply:
why comment import lines?

Reply:
It is meant to be addressed at the top of your javadoc comment. Just add in a description (overview) of what this class is used for and the content it holds.

Reply:
but you could just look at the class, that particular ENUM is not anything special.

Reply:
Fixed in commit 0872604